data/dog_d.json

```diff
@@ -0,0 +1,171 @@
+{
+  "name": "DOG D",
+  "tagline": "ANNIVERSARY IMPERIAL STOUT.",
+  "first_brewed": "05/2015",
+  "description": "Dog D is an extra special brew, marking 8 years of BrewDog. Like Dog A, B & C before it, Dog D is an Imperial Stout with speciality ingredients, only this time we’ve barrel-aged it too. Dog D pours like dark matter into your glass, with a deep tan head. The nose is subtle but powerful, and on the palate, there’s a treacle toffee backbone, supporting the spicy hop flavour and naga chilli heat.",
+  "abv": 16.1,
+  "ibu": 85,
+  "target_fg": 1015,
+  "target_og": 1125,
+  "ebc": 400,
+  "srm": 200,
+  "ph": 4.3,
+  "attenuation_level": 84,
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": {
+      "temp": {
+        "value": 63,
+        "unit": "celsius"
+      },
+      "duration": 90
+    },
+    "fermentation": {
+      "temp": {
+        "value": 21,
+        "unit": "celsius"
+      }
+    },
+    "twist": "Add 1 naga chilli in the FV"
+  },
+  "ingredients": {
+    "malt": [
+      {
+        "name": "Pale Ale",
+        "amount": {
+          "value": 7.5,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Wheat",
+        "amount": {
+          "value": 0.5,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Flaked Oats",
+        "amount": {
+          "value": 0.75,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Dark Crystal",
+        "amount": {
+          "value": 0.75,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Carafa Special Malt Type 1",
+        "amount": {
+          "value": 0.5,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Carafa Special Malt Type 3",
+        "amount": {
+          "value": 0.5,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Rye",
+        "amount": {
+          "value": 0.75,
+          "unit": "kilograms"
+        }
+      },
+	  {
+        "name": "Brown",
+        "amount": {
+          "value": 0.75,
+          "unit": "kilograms"
+        }
+      }
+    ],
+    "hops": [
+      {
+        "name": "Chinook",
+        "amount": {
+          "value": 50,
+          "unit": "grams"
+        },
+        "add": "Start",

```

Comment:
Could you lowercase this word please, just to keep consistent with the other recipes :smiley: